playbooks/HU.json

```diff
@@ -0,0 +1,33 @@
+{
+  "country": {
+    "iso": "HU",
+    "name": "Hungary"
+  },
+  "source": "HungaroControl AIS",
+  "scraper": {
+    "engine": "axios",
+    "features": {
+      "baseUrl": "https://ais-en.hungarocontrol.hu/aip/2021-06-17/",
```

Comment:
The latest AIP is `https://ais-en.hungarocontrol.hu/aip/2021-08-12/`

Reply:
Ah you are right, wrong one